app/src/main/java/net/tensory/rxjavatalk/house/HousePresenter.java

```diff
@@ -70,7 +68,7 @@ private HousePresenter(House house,
     private void updateFromBattles(House house, HouseAssetProfileProvider assetProfileProvider, Battle battle) {
         battle.getHouseBattleResults()
                 .stream().
-                filter(combatant -> house == combatant.getHouse())
+                filter(combatant -> house.equals(combatant.getHouse()))
```

Comment:
This change can totally stay, but just wanted to let you know that `==` is completely safe for Enums, since they are static.  `.equals` just calls`==` under the hood

Reply:
Today I learned! Thank you!